### PR TITLE
Make alternate main screens optional

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary

SUMMARY: Interface "Make seasonal screen and optional and revert last title screen change"

#### Purpose of change

- Make seasonal screen optional
- Bring back vanilla title screen
- Move alternate title screen to new file

#### Describe the solution

Add an option for seasonal screen
Bring back vanilla title screen
Move alternate title screen to new file
Add option to set frequency of appearance of the alternate title screen
Set the default frequency to 10%

#### Testing

image

image

Open and close option menu several times > get both possible title screens

/!\ I was not able to test the russian screen

### CODE REVIEW

File: data/title/ru.title [hunk 1]
```diff
@@ -13,13 +13,6 @@
...
-<color_white>    *   *  /-\                                       </color><color_light_green>,,,,/   |           |    \ </color>
-<color_white>     *** =========='      </color><color_red>----               ----        </color><color_light_green>`  _/`'.,       \ _--  </color>
-<color_white>     ||'''/ |                                        </color><color_light_green>`````'-    /```''-    | |  </color>
-<color_white>     ,\.    |                                                </color><color_light_green>'.-_`,,,,,,,,,\.\--</color>
-<color_white>    /  |    ^          ___,,,,,,,,,,...-------''''''``````````                  </color>
-<color_brown>*------'''''*``````````                                                         </color>
\ No newline at end of file
+<color_light_blue>            |    |)  \ |  Y  |\  ___/ |  |)  >\  ___/ _/ /\ \_|  /  |           </color>
+<color_light_blue>            |______  / |__|  / \___  >|   __/  \___  >\  __  /\__/  /           </color>
+<color_light_blue>                   \/      \/      \/ |__|         \/  \/  \/     \/            </color>
```
Comment (ZhilkinSerg) [inline data/title/ru.title:18]:
```suggestion
<color_light_blue>                   \/      \/      \/ |__|         \/  \/  \/     \/            </color>

```

**Comment by Zireael07** [comment]
What is the reasoning behind the revert part?

File: src/path_info.cpp [hunk 2]
```diff
@@ -10,6 +10,7 @@
 #if defined(_WIN32)
 #include <windows.h>
 #endif
+#include <visitable.cpp>
```
Comment (ZhilkinSerg) [inline src/path_info.cpp:13]:
```suggestion
#include "visitable.cpp"
```

**Comment by ZhilkinSerg** [comment]
> 
> 
> What is the reasoning behind the revert part?

Some people don't like this art it for various reasons.

It took me some time to copy from Chinese title screen, so I would not simply revert title screen to classic, but rather move title with picture to a separate file and show this file randomly (one in X) when seasonal title screens are on.

File: data/title/ru.alt1 [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+<color_light_blue>            |______  / |__|  / \___  >|   __/  \___  >\  __  /\__/  / </color><color_light_green>_  </color><color_light_green>|   `  </color>
+<color_light_blue>                   \/      \/      \/ |__|         \/  \/  \/     \/   </color><color_light_green>',|    \ </color>
+<color_white>     ***                                                  </color><color_light_green>_..        -.,,'-., / </color>
+<color_white>    *   *  /-\                                       </color><color_light_green>,,,,/   |           |    \ </color>
+<color_white>     *** =========='      </color><color_red>----               ----        </color><color_light_green>`  _/`'.,       \ _--  </color>
+<color_white>     ||'''/ |                                        </color><color_light_green>`````'-    /```''-    | |  </color>
+<color_white>     ,\.    |                                                </color><color_light_green>'.-_`,,,,,,,,,\.\--</color>
+<color_white>    /  |    ^          ___,,,,,,,,,,...-------''''''``````````                  </color>
+<color_brown>*------'''''*``````````                                                         </color>
```
Comment (ZhilkinSerg) [inline data/title/ru.alt1:25]:
```suggestion
<color_brown>*------'''''*``````````                                                         </color>

```

**Pull request merged**